src/Core/DB/Adapter.php

```diff
@@ -86,6 +86,13 @@ public function errno(): ?string;
      */
     public function error(): ?string;
 
+    /**
+     * Returns MySQL-styled error code (.....)
```

Comment:
Why MySQL-styled?

Reply:
https://www.php.net/manual/ru/pdo.errorcode.php

Reply:
There is nothing about MySQL

Reply:
Will rename to SQLSTATE